src/subprocess-win32.cc

```diff
@@ -74,7 +74,52 @@ HANDLE Subprocess::SetupPipe(HANDLE ioport) {
   return output_write_child;
 }
 
-bool Subprocess::Start(SubprocessSet* set, const string& command) {
+static DWORD POSIXPriorityToWin32(int unixPriority) {
+  if (unixPriority <= -20)
+    return REALTIME_PRIORITY_CLASS;
+
+  switch (unixPriority) {
+  case -19:
+  case -18:
+  case -17:
+  case -16:
+  case -15:
+  case -14:
+  case -13:
+  case -12:
+  case -11:
+  case -10:
+    return HIGH_PRIORITY_CLASS;
+  case -9:
+  case -8:
+  case -7:
+  case -6:
+  case -5:
+  case -4:
+  case -3:
+  case -2:
+  case -1:
+    return ABOVE_NORMAL_PRIORITY_CLASS;
+  case 0:
+    return NORMAL_PRIORITY_CLASS;
+  case 1:
+  case 2:
+  case 3:
+  case 4:
+  case 5:
+  case 6:
+  case 7:
+  case 8:
+  case 9:
+  case 10:
+    return BELOW_NORMAL_PRIORITY_CLASS;
+  default:
+    return IDLE_PRIORITY_CLASS;
+  }
```

Comment:
Can you change this to use normal range checks instead for readability?

Reply:
Done